airflow/models/__init__.py

```diff
@@ -688,6 +632,57 @@ def __init__(self, task, execution_date, state=None):
         # Not persisted to the database so only valid for the current process
         self.raw = False
 
+    @staticmethod
+    @provide_session
+    def clear_task_instances(tis,
+                             activate_dag_runs=True,
+                             dag=None,
+                             session=None
+                             ):
+        """
+        Clears a set of task instances, but makes sure the running ones
+        get killed.
+
+        :param tis: a list of task instances
+        :param session: current session
```

Comment:
The docstring ordering isn't correct, can we move the session to the end?

Reply:
fixed